ui/src/components/ClientScopesTable/index.jsx

```diff
@@ -113,18 +115,18 @@ export default class ClientScopesTable extends Component {
       map(node => (
         <TableRow key={node}>
           <TableCell padding="dense">
-            <Link
-              className={classes.tableCell}
+            <TableCellItem
+              dense
+              button
+              component={Link}
               to={
                 selectedScope
                   ? `/auth/clients/${encodeURIComponent(node)}`
                   : `/auth/scopes/${encodeURIComponent(node)}`
               }>
-              <div className={classes.listItemCell}>
-                <Typography>{node}</Typography>
-                <LinkIcon size={16} />
-              </div>
-            </Link>
+              <ListItemText primary={<Typography>{node}</Typography>} />
```

Comment:
```suggestion
              <ListItemText primary={node} />
```

Reply:
Then we could also stop importing the `Typography` component.

Reply:
ok thank you